mapper/types.go

```diff
@@ -13,11 +13,6 @@ const (
 	FujiChainID = 43113
 	FujiAssetID = "U8iRqJoiJm8xZHAacmvYyZVwqQx6uDNtQeP3CQ6fcgQk3JqnK"
 
-	TokenTypeMetadata        = "tokenType"
```

Comment:
Why are we removing constants?

Reply:
`TokenTypeMetadata` and `TokenSymbol` are unused. For metadata maps, we haven't used constants elsewhere in the codebase as it sometimes creates circular dependencies.

I can put them back if you'd like but seemed odd to make an exception for just these two